mono_repo/lib/src/commands/travis.dart

```diff
@@ -75,7 +95,17 @@ void _writeTravisScript(
     stderr.writeln(yellow.wrap('  chmod +x $travisShPath'));
   }
 
-  travisScript.writeAsStringSync(_travisSh(taskEntries, pkgEntries));
+  var packageCase = pkgEntries.isEmpty
```

Comment:
minor nit: pull this out into a helper method. 

This method doesn't contain any other code to actually generate the content inside the travis script, just the bits of actually writing the file.